presto-product-tests/src/main/resources/sql-tests/testcases/tpcds/q85.result

```diff
@@ -1,7 +1,7 @@
--- delimiter: |; types: CHAR|DOUBLE|DECIMAL|DECIMAL
-Gift exchange                                                                                       |76.00000000000|464.36|8.62|
-Not the product that                                                                                |70.00000000000|876.67|46.67|
-Parts missing                                                                                       |7.00000000000|129.42|38.65|
-reason 23                                                                                           |47.00000000000|734.61|6.17|
-reason 25                                                                                           |5.00000000000|48.94|53.15|
-reason 28                                                                                           |8.00000000000|306.20|37.06|
+-- delimiter: |; types: VARCHAR|DOUBLE|DECIMAL|DECIMAL
+Gift exchange       |76.00000000000|464.36|8.62|
```

Comment:
Do you need trailing spaces here? If not please remove them.

Reply:
These are needed